src/Settings/NumberGeneratorOptions.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import { stripesConnect, withStripes } from '@folio/stripes/core';
+import { ConfigManager } from '@folio/stripes/smart-components';
+
+import { Col, RadioButton, Row } from '@folio/stripes/components';
+
+const NumberGeneratorOptions = (props) => {
+  const ConnectedConfigManager = stripesConnect(ConfigManager);
```

Comment:
any reason to call HoC inside a component? use `stripes.connect`, there are many examples

Reply:
Was following example set by ui-users. Easy fix though, next commit will resolve